airflow/providers/amazon/aws/operators/eks.py

```diff
@@ -59,7 +59,54 @@
 FARGATE_FULL_NAME = "AWS Fargate profiles"
 
 
-class EksCreateClusterOperator(BaseOperator):
```

Comment:
Dont we need deprecation warning here, and call the new operator if we remove this class? Otherwise existing DAGs using the `EksCreateClusterOperator` will break right?

Reply:
I'm not removing the class, just making it inherit from a different class. It shouldn't affect any existing DAGs.

Reply:
Yes Github makes it confusing but look at line `107`, `EksCreateClusterOperator` is still there